### PR TITLE
ci: spin up blind-csp using docker-compose

### PR DESCRIPTION
**Pull request by altergui**
env var BLINDCSP_URL is passed to "yarn test" step

### CODE REVIEW

**Comment by altergui** [comment]
fixes #70

**Review by marcvelmer — COMMENTED** [review]
I also need the `BLINDCSP_PUBKEY` for creating the election

**Comment by altergui** [comment]
ok let me add it

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -63,11 +63,19 @@ jobs:
...
           echo -n "voconed_hostport=" >> $GITHUB_ENV
           docker-compose -f test/integration/util/docker-compose.yml port voconed 9095 >> $GITHUB_ENV
+          echo -n "blindcsp_hostport=" >> $GITHUB_ENV
+          docker-compose -f test/integration/util/docker-compose.yml port blind-csp 5000 >> $GITHUB_ENV
 
       - name: Integration tests
         run: yarn test:integration --ci --coverage --maxWorkers=2
         env:
           API_URL: http://${{ env.voconed_hostport }}/v2
+          BLINDCSP_URL: http://${{ env.blindcsp_hostport }}/v1/auth/elections
```
[the remark this answers is not in the document]
Comment (altergui) [reply .github/workflows/main.yml:73]: i believe the README is outdated or inaccurate: baseURL is `/v1/auth/elections` 

```
$ docker run vocdoni/blind-csp:latest --help
pflag: help requested
Usage of /app/blind-csp:
      --baseURL string        base URL path for serving the API (default "/v1/auth/elections")
```

and looking at the code there's no mention of `/v1/auth/processes` (except in comments). the daemon registers `/v1/auth/elections/{processId}/{signType}/sign`
```
2023-02-07T10:25:19Z	INFO	httprouter/httprouter.go:220	added public handler for namespace bearerStd with pattern /v1/auth/elections/{processId}/{signType}/sign
```

[the remark this answers is not in the document]
Comment (altergui) [reply .github/workflows/main.yml:73]: pushed a PR fixing the README :raised_hands:

[the remark this answers is not in the document]
Comment (altergui) [reply .github/workflows/main.yml]: also changed BLINDCSP_URL to `http://${{ env.blindcsp_hostport }}/v1` :grin:

**Pull request merged**